.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- Provide a general summary of your changes in the title above
+
+Please read our contribution guidelines and sign the Contributor License
+Agreement (CLA) before submitting the pull request. Also, check if there are no
+other open pull requests targeting the same issue. -->
+
+#### Description
+<!-- Describe your changes in detail. -->
+
+##### Category
+<!-- Tell us what type of issue does your pull request target.
+You can uncomment one of the following options: -->
+
+<!-- Bug Fix<br> -->
+<!-- New Feature<br> -->
+<!-- Improvement<br> -->
+
+##### Relevant issue
+<!-- Provide a link to the related issue. You can use the following keywords
+and the issue number: "fixes", "resolves", "relates to". Eg : closes #21
+
+We accept only pull request related to open issues.If you're suggesting a new
+feature, change or fixing a bug that is not yet reported, please discuss it in
+an issue before submitting a pull request. -->
+
+#### Testing
+<!-- Tell us how you have tested the changes. -->
+
+##### Steps to run the tests
+<!-- Describe a set of steps to run the tests relevant to this change. -->
+
+1. 
+2. 
+3.  
+
+#### Checklist 
+<!-- Please check if the pull request fulfils these requirements: -->
+
+- [ ] Name is added to the NOTICE file, if it is not present already.
+- [ ] Changes are rebased over the latest commit of the target branch.
```

Comment:
```suggestion
- [ ] Changes are rebased onto the target branch.
```

Reply:
Addressed in this commit - 1c2dcf9403c97c826b7fa701b7398c9a3a379363.